project.yml

```diff
@@ -0,0 +1,354 @@
+    - evaluate-parser-tagger-md
+    - evaluate-parser-tagger-lg
+    - evaluate-ner-sm
+    - evaluate-ner-md
+    - evaluate-ner-lg
+    - evaluate-specialized-ner
+
+commands:
+  - name: download
+    help: "Download the necessary files"
+    script:
+      - "aws s3 cp s3://ai2-s2-scispacy/data/gorc_subset.freqs assets/gorc_subset.freqs --no-sign-request"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/pubmed_with_header.txt.gz assets/pubmed_with_header.txt.gz --no-sign-request"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/genia/train.json assets/genia_train.json --no-sign-request"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/genia/dev.json assets/genia_dev.json --no-sign-request"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/genia/test.json assets/genia_test.json --no-sign-request"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/en-core-web.tgz assets/en-core-web.tgz"
+      - "tar -xzvf assets/en-core-web.tgz -C assets/"
+      - "rm assets/en-core-web.tgz"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/med_mentions.tar.gz assets/med_mentions.tar.gz"
+      - "tar -xzvf assets/med_mentions.tar.gz -C assets/"
+      - "rm assets/med_mentions.tar.gz"
+      - "aws s3 cp s3://ai2-s2-scispacy/data/ner/ assets --recursive --exclude '*' --include '*.tsv'"
+    outputs:
+      - "assets/gorc_subset.freqs"
+      - "assets/pubmed_with_header.txt.gz"
+      - "assets/genia_train.json"
+      - "assets/genia_dev.json"
+      - "assets/genia_test.json"
+      - "assets/en-core-web/train"
+      - "assets/en-core-web/dev"
+      - "assets/en-core-web/test"
+      - "assets/corpus_pubtator.txt"
+      - "assets/BC5CDR-IOB/train.tsv"
+      - "assets/BC5CDR-IOB/devel.tsv"
+      - "assets/BC5CDR-IOB/test.tsv"
+      - "assets/JNLPBA-IOB/train.tsv"
+      - "assets/JNLPBA-IOB/devel.tsv"
+      - "assets/JNLPBA-IOB/test.tsv"
+      - "assets/CRAFT-IOB/train.tsv"
+      - "assets/CRAFT-IOB/devel.tsv"
+      - "assets/CRAFT-IOB/test.tsv"
+      - "assets/BioNLP13CG-IOB/train.tsv"
+      - "assets/BioNLP13CG-IOB/devel.tsv"
+      - "assets/BioNLP13CG-IOB/test.tsv"
+
+  - name: convert-shared
+    help: "Convert the data to spaCy's format"
+    script:
+      - "spacy convert assets/genia_train.json project_data --converter json --file-type spacy"
+      - "spacy convert assets/genia_dev.json project_data --converter json --file-type spacy"
+      - "spacy convert assets/genia_test.json project_data --converter json --file-type spacy"
+      - "spacy convert assets/en-core-web project_data --converter json --file-type spacy"
+    deps:
+      - "assets/genia_train.json"
+      - "assets/genia_dev.json"
+      - "assets/genia_test.json"
+      - "assets/en-core-web"
+    outputs:
+      - "project_data/genia_train.spacy"
+      - "project_data/genia_dev.spacy"
+      - "project_data/genia_test.spacy"
+      - "project_data/train"
+      - "project_data/dev"
+      - "project_data/test"
+
+  - name: convert-sm
+    help: "Convert the data to spaCy's format"

```

Comment:
How about - "convert a file of word/document frequency statistics to spacy's vocabulary format."